MAINTAINERS.md

```diff
@@ -0,0 +1,13 @@
+# Maintainers 👔
+
+Tips & tricks for maintainers of Tickety-Tick.
+
+## Sign add-on for Firefox
+
+Offering a (beta-) version for download requires self-signing the add-on through [addons.mozilla.org](https://extensionworkshop.com/documentation/publish/#distribute-your-signed-extension). This generates a `.xpi` for installing the extension in Firefox.
+
+Find the API credentials in 1Password and run:
+
+```shell
+WEB_EXT_API_KEY=xxx WEB_EXT_API_SECRET=xxx yarn build:firefox && yarn sign
```

Comment:
I think you'll probably want to set the environment variables on the second command.

```suggestion
yarn build:firefox && WEB_EXT_API_KEY=xxx WEB_EXT_API_SECRET=xxx yarn sign
```

To illustrate:

```sh
~ ❯ cat > try 
#!/usr/bin/env sh
echo $VAR
^C

~ ❯ chmod a+x try

~ ❯ VAR=yep ./try && ./try
yep

```

You should see 2 "yep"s here.

Reply:
🤹 nice! I already had a vague feeling it may behave like that.